PocketKit/Sources/SharedPocketKit/Migrations/Cleanup/CleanupService.swift

```diff
@@ -0,0 +1,87 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+public enum CleanupError: Error {
+    case fileNotFound
+}
+
+/// Service that cleans up legacy data
+public struct LegacyCleanupService {
+    public func cleanUp() {
+        cleanupLegacyUserDefaults()
+        deleteSqliteDataBase()
+        deleteImageCache()
+        deleteLogFiles()
```

Comment:
should we also add `deleteAudioCache` here as well? Or will it be handled somewhere else, if so why?

Reply:
indeed we should @cyndichin , thanks for catching this!

Reply:
np! seems like you have 2 approvals already, but can re-review when conflicts addressed